packages/commonwealth/client/scripts/views/components/component_kit/cw_cover_image_uploader.tsx

```diff
@@ -280,7 +280,7 @@ export const CWCoverImageUploader = ({
 
     pseudoInput.current.addEventListener('change', pseudoInputHandler);
     attachZone.current.addEventListener('click', (e: any) => {
-      if (e.target.classList.contains('attach-zone')) clickHandler(e);
+      if (e.target.classList.contains('attach-btn')) clickHandler(e);
```

Comment:
we should include both 'attach-zone' and 'attach-btn' to cover the whole clickable area to trigger OS file explorer

Reply:
@mzparacha Just made that change, thanks for letting me know